src/m_cheat.c

```diff
@@ -163,10 +163,10 @@ struct cheat_s cheat[] = {
   {"idbeholdr",  "Radiation Suit",    not_net | not_demo,
    {cheat_pw}, pw_ironfeet },
 
-  {"idbeholda",  "Auto-map",          not_net | not_demo,
+  {"idbeholda",  "Auto-map",          not_dm,
```

Comment:
So it is considered okay nowadays to cheat for a complete automap or full sight while recording a demo? @kraflab ?

Reply:
Okay, we don't have any checks for strict mode wrt cheats yet, right?

Reply:
> but it should still be impossible to cheat in a netgame.

Even in co-op? I thought `not_dm` was sufficient.